### DISCUSSION TITLE
ALTER SYSTEM DROP BACKEND报错？

### DISCUSSION THREAD

**Discussion by a3225167** [apache/doris#12173] [A - General / Q&A] [answered]
无法删除BACKEND，
ALTER SYSTEM DROP BACKEND  '192.168.191.108:9050';
<img width="1550" alt="image" src="https://user-images.githubusercontent.com/32891043/187343112-47658eff-1413-4e4a-855a-ab763cf72fe4.png">

**Comment by LemonLiTree** [comment, marked as answer]
ALTER SYSTEM DROPP BACKEND '192.168.191.108:9050';   用DROPP

**Comment by dataalive** [comment]
more safe way is using decommission https://doris.apache.org/docs/dev/admin-manual/cluster-management/elastic-expansion#delete-be-nodes
